pontoon/actionlog/models.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from pontoon.base.models import (
+    Entity,
+    Locale,
+    Translation,
+    User,
+)
+
+
+class ActionLog(models.Model):
+    ACTIONS_TYPES = (
+        # A translation has been created.
+        ('translation:created', 'Translation created'),
+
```

Comment:
There are three different actions in in `pontoon.base.views:update_translation()` (add first translation, add additional translation, update translation). We should log all three of them, possibly as 2 or 3 different types.

Reply:
What's "update translation"? Is that even possible? Why should we differentiate between adding the first translation to an entity and adding a non-first translation?

Reply:
Update happens when you unfuzzy or approve a suggestion by submitting identical translation:
https://github.com/mozilla/pontoon/blob/3d5d5130309b4119313aa4af7cdc907d6bd7d937/pontoon/base/views.py#L552

We don't need to differentiate between first/non-first translations. But we should log all 3 actions in `pontoon.base.views:update_translation()`.